### PR TITLE
Closes #825: Adjust scss theme file to support high contrast version

### LINKED ISSUE #825: Adjust scss theme file to support high contrast version

### PR DESCRIPTION
**Pull request by sudmar**
Closes #825

Notes:
- font file was updated to include monocolor Dataverse logos and icons, please hit Ctrl+F5 to refresh
- footer logos and statistic charts are not changed in the contrast versions
- you need to add class `high-contrast-yellow-on-black` to the `<body>` tag, or modify the `3524` line in the `theme.scss` file for testing purposes (so it's always active)
- there are also a few other high contrast themes present in the `.scss` file which are mostly complete, but not fully tested. They're commented out for now.

### CODE REVIEW

File: dataverse-webapp/src/main/webapp/resources/vecler/theme.scss [hunk 1]
```diff
@@ -2454,7 +2514,1084 @@ div.ui-dialog.largePopUp {
...
+			border-right-color: $contrast-foreground-color;
+		}
+	}
+}
+
+/* Definitions */
+
+body.high-contrast-yellow-on-black {
+	@include high-contrast-template();
+}
```
Comment (madryk) [inline dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Link for clearing facet value is not readable
clear_facet_link

Comment (madryk) [inline dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: There is problem with focus styling on checkboxes when they are unselected (in many places)
checkbox_focus_when_unselected2
In above example I have focus on "Astronomia i Astrofizyka"

Comment (madryk) [inline dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Presenting terms of use on account create is styled wrong 
terms_of_use

Comment (madryk) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Edit menu is not visible when file is selected in datatable
datatable_selected_menu

Comment (madryk) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Disabled links on pagination in search page have to low contrast ratio
paging_disabled_links

Comment (madryk) [inline dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: I think that on widget with autocomplete - no search result info should be in some kind of border
autocomplete_styling

Comment (madryk) [inline dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Sending request for access to file have bad styled panel
request_access_dialog

Comment (madryk) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Right now we can't tell the difference between input text and its placeholder
placeholders

Comment (madryk) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: fileupload errors are not styled:
fileupload_error_box

Comment (madryk) [inline dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Problems with datepicker theme:
datepicker_styling

Comment (madryk) [inline dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: changing page size widget 
files_paging_size
 in files datatable should probably have some border

Comment (madryk) [inline dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Are we able to add some border for tooltips? Sometimes they blends with other components
tooltip_and_background

Comment (madryk) [inline dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: I'm not sure - but maybe we should indicate somehow readonly textareas
readonly_textbox

Comment (madryk) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: IMO - we should have perceptible change in styling when mouse is over arrow
featured_arrow

Comment (madryk) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: we have problems with style for selectcheckboxmenu in edit metadata
edit_metadata_subject

**Review by madryk — CHANGES_REQUESTED** [review]

File: dataverse-webapp/src/main/webapp/resources/vecler/theme.scss (hunk 1 above)
Comment (sudmar) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Fixed

Comment (sudmar) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Fixed

Comment (sudmar) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Fixed

Comment (sudmar) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Fixed

Comment (sudmar) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Fixed

Comment (sudmar) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Fixed

Comment (sudmar) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Fixed

Comment (sudmar) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Fixed

Comment (sudmar) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:3676]: Fixed

File: dataverse-webapp/src/main/webapp/resources/vecler/theme.scss [hunk 2]
```diff
@@ -2454,7 +2529,1175 @@ div.ui-dialog.largePopUp {
...
+
+		.text-danger {
+			color: $contrast-foreground-color;
+		}
+	}
+
+	.ui-autocomplete-emptyMessage {
+		border-color: $contrast-foreground-color;
+		background: $contrast-background-color;
+		color: $contrast-foreground-color;
```
Comment (madryk) [inline dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:2850]: Border for empty autocomplete is still not showing up
autocomplete_border_1
maybe we should add `border-style: solid;`?
autocomplete_border

File: dataverse-webapp/src/main/webapp/resources/vecler/theme.scss [hunk 3]
```diff
@@ -2454,7 +2529,1175 @@ div.ui-dialog.largePopUp {
...
-
+  
 .no-bottom-border {
     border-bottom: none !important;
 }
+
+
+/**
+ * HIGH CONTRAST VERSION BELOW
+ */
```
Comment (madryk) [inline dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:2544]: I have some issues with submenu for selected file in datatable
menu_selected_file_tabular

**Review by madryk — CHANGES_REQUESTED** [review]

File: dataverse-webapp/src/main/webapp/resources/vecler/theme.scss (hunk 2 above)
Comment (sudmar) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:2850]: Yeah, I overlooked that. Fixed

File: dataverse-webapp/src/main/webapp/resources/vecler/theme.scss (hunk 3 above)
Comment (sudmar) [reply dataverse-webapp/src/main/webapp/resources/vecler/theme.scss:2544]: Fixed

**Review by madryk — APPROVED** [review]

**Pull request merged**